python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -350,6 +358,19 @@ async def do_takeImages(self, id_data):
             exposure_time = id_data.expTime
             time_stamp = time.time()
 
+            # Calculate expected time for IN_PROGRESS ack
+            SHUTTER_TIME = (
```

Comment:
Usually `UPPER_SNAKE_CASE`  names are reserved for constants. These here seems to be normal variables, I suggest using the regular `lower_snake_case`.

Reply:
Done. That was a rookie mistake.